### ISSUE TITLE
Enable/Disable Platform

### ISSUE THREAD

**Issue by HoeblingerDaniel** [TcNobo/TcNo-Acc-Switcher#32]
As the count of platforms is growing it would be nice to have the ability to disable platforms on the startup screen to reduce clutter.

**Comment by HoeblingerDaniel** [comment]
Also I would like to make a Pull Request in the near future implementing this.
I plan to do it by just adding a bool enabled to each platforms settings and having a small gear symbol in the bottom right to open a menu where you can enable/disable them.

**Comment by TcNobo** [comment]
I'd like to add a settings screen to the main menu to toggle just the shared options.
I'm planning on adding a right-click menu to:
- Hide a platform ("Hide Platform")
- Show a list of hidden platforms, though this would be on the settings menu mentioned earlier, not so important for now. ("Show hidden platforms")
- Enable reorganising, so users can drag them around similar to the account reorganisation.

tldr: Yes, you can add the "Hide Platform" option. If possible do it through a context menu, similar to those on the existing account lists. I'll reorganise things later ~

**Comment by TcNobo** [comment]
Merged.
